Foundation/IndexPath.swift

```diff
@@ -10,6 +10,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+import CoreFoundation
```

Comment:
free is not defined in CoreFoundation, it is defined in either Darwin or Glibc so the import should be of one of those.